src/utils/validateDomainName.ts

```diff
@@ -7,3 +7,9 @@ export default function validateDomainName(domain: string): string | void {
     if (domain === "") return null;
     if (!DOMAIN_NAME_REGEX.test(domain)) return "Domain name is not valid";
 }
+
+export function validateRequiredHostName(domain: string): string | void {
+    // for optional fields to work
+    if (domain === "") return "Host name is required";
+    if (!DOMAIN_NAME_REGEX.test(domain)) return "Host name is not valid";
+}
```

Comment:
Please use `validateDomainName` here instead of re-implement it

Reply:
ValidateDomainName has a different error --> Domain name instead of Host name